### PR TITLE
Npm tweaks for Team City

### PR DESCRIPTION
**Pull request by rich-nguyen**
This is a mix of analysis work so that we can better audit where the time is spent in Team City, and an npm shrinkwrap fix.

I've split the bash script we use to package our assets and code into three smaller files, one for npm, another grunt's asset compilation, and another for sbt play packaging. We can then use Team Cit's perf tools to see how long each step is taking.

also, @sndrs has modified our npm package.json, but the packages are still being downloaded by everyone because of shrinkwrap. I am still not sure what's best regarding shrinkwrapping. I would hope a semver solves most issues, and i am looking for a way to solve a transitive dependency issue.

### CODE REVIEW

**Comment by desbo** [comment]
:+1:

File: package.json [hunk 1]
```diff
@@ -56,7 +56,7 @@
     "karma-script-launcher": "0.1.0",
     "karma-spec-reporter": "0.0.19",
     "load-grunt-config": "0.17.1",
-    "megalog": "^0.0.2",
+    "megalog": "^0.1.0",
```
Comment (rich-nguyen) [inline package.json]: upgrading megalog @sndrs :)

**Comment by rich-nguyen** [comment]
Example team city perf log for breakdown of tasks:

screen shot 2015-08-12 at 18 26 20

File: package.json (hunk 1 above)
Comment (sndrs) [inline package.json]: :seedling:

**Comment by sndrs** [comment]
:+1:

**Pull request merged**